internal/bootstrap/bootstrap_tpl.go

```diff
@@ -272,12 +266,7 @@ const bootstrapTemplate = `{
       "api_type": "DELTA_GRPC",
       "transport_api_version": "V3",
       "grpc_services": {
-        "initial_metadata": [
-          {
-            "key": "x-consul-token",
-            "value": "{{ .Token }}"
-          }
-        ],
```

Comment:
since this is generated code, worth implementing the hackery in the makefile helper : https://github.com/hashicorp/consul-dataplane/blob/main/Makefile#L80